backend/FwLite/FwDataMiniLcmBridge/Api/FwDataMiniLcmApi.cs

```diff
@@ -1331,6 +1331,12 @@ internal void CreateExampleSentence(ILexSense lexSense, ExampleSentence exampleS
             lexExampleSentence.Reference.get_WritingSystem(0));
     }
 
+    public ICmTranslation CreateExampleSentenceTranslation(ILexExampleSentence parent)
```

Comment:
Perhaps `CreateExampleSentence` should use this method. It has exactly the same code.

Reply:
Done in commit 13d51731. Yes, that's a good idea to avoid any possible code divergence in the future.